acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/node/PantheonNode.java

```diff
@@ -219,21 +230,14 @@ private String createPantheonContainer(final NodeConfiguration config) {
               "--rpc-http-enabled",
               "--rpc-ws-enabled",
               "--rpc-http-apis",
-              "ETH,NET,WEB3,IBFT,MINER,ADMIN,EEA");
+              "ETH,NET,WEB3,IBFT,MINER,ADMIN,EEA",
```

Comment:
does this not cause the "METHOD_NOT_ENABLED" test to fail?

Reply:
It should but it doesn't. So if you have IBFT enabled but aren't using IBFT then Pantheon doesn't consider it enabled.
But since we don't need IBFT in our ATs i'm removing it from the list so it's clearer.